core/src/test/java/org/apache/brooklyn/core/catalog/internal/SpecParameterInMetaTest.java

```diff
@@ -43,18 +49,40 @@
 public class SpecParameterInMetaTest {
     private ManagementContext mgmt;
     private BrooklynCatalog catalog;
-    private String spec;
+    private String specId;
 
     @BeforeMethod(alwaysRun=true)
     public void setUp() {
         mgmt = LocalManagementContextForTests.newInstanceWithOsgi();
         catalog = mgmt.getCatalog();
-        spec = TestToSpecTransformer.registerSpec(EntitySpec.create(BasicEntity.class));
+        StaticTypePlanTransformer.forceInstall();
+        PlanToSpecFactory.forceAvailable(TestToSpecTransformer.class, JavaCatalogToSpecTransformer.class);
```

Comment:
btw removing parameters in meta support made this class obsolete, see https://github.com/apache/incubator-brooklyn/pull/1036. Some other conflicts in the tests between the PRs unfortunately due to a renaming which was needed.